### PR TITLE
Server Management swagger description edits

### PR DESCRIPTION
**Pull request by steved0x**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/716)

### CODE REVIEW

File: arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json [hunk 1]
```diff
@@ -1923,68 +1923,68 @@
...
         "minorRevision": {
           "type": "integer",
-          "description": "the LSW of the fourth part"
+          "description": "The LSW of the fourth part."
         }
       }
     },
     "PowerShellSessionResources": {
-      "description": "a collaction of powershell session resources",
+      "description": "A collaction of powershell session resources.",
```
Comment (BryanLa) [inline arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json]: Typo in "collaction"

File: arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json [hunk 2]
```diff
@@ -1923,68 +1923,68 @@
...
       "type": "object",
       "properties": {
         "value": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/PowerShellSessionResource"
           },
-          "description": "Colleciton of powershell session resources"
+          "description": "Colleciton of powershell session resources."
```
Comment (BryanLa) [inline arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json]: Typo in "Colleciton"

File: arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json [hunk 3]
```diff
@@ -2105,7 +2105,7 @@
       }
     },
     "PowerShellTabCompletionResults": {
-      "description": "an array of strings representing the different values that can be tabbed thru",
+      "description": "An array of strings representing the different values that can be tabbed through.",
```
Comment (BryanLa) [inline arm-servermanagement/2015-07-01-preview/swagger/servermanagement.json]: Given that we don't know the client, would "selected" be better than "tabbed through"?

File: arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json [hunk 4]
```diff
@@ -82,19 +82,19 @@
...
           "201": {
-            "description": "the operation completed successfully",
+            "description": "The operation completed successfully.",
             "schema": {
               "$ref": "#/definitions/GatewayResource"
             }
           },
           "default": {
-            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema, and an exception will be thrown.",
```
Comment (BryanLa) [inline arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json]: Typo in "defintion"

File: arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json [hunk 5]
```diff
@@ -259,13 +259,13 @@
...
           "200": {
-            "description": "the operation completed successfully.",
+            "description": "The operation completed successfully.",
             "schema": {
               "$ref": "#/definitions/GatewayResource"
             }
           },
           "default": {
-            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema, and an exception will be thrown.",
```
Comment (BryanLa) [inline arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json]: Typo in "defintion". I'm not going to do any more comments for these, so you might want to run spell check on the entire file.

File: arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json [hunk 6]
```diff
@@ -437,13 +437,13 @@
...
           "200": {
-            "description": "the operation completed successfully."
+            "description": "The operation completed successfully."
           },
           "202": {
             "description": "Accepted: Location header contains the URL where the status of the long running operation can be checked."
           },
           "default": {
-            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema, and an exception will be thrown.",
```
Comment (BryanLa) [inline arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json]: Should "Error" be capitalized? (there are several instances of this)

File: arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json [hunk 7]
```diff
@@ -1663,48 +1663,48 @@
...
         },
         "secondaryEncryptionCertificateThumbprint": {
           "type": "string",
-          "description": "Secondary thumbprint of the encryption certificate"
+          "description": "Secondary thumbprint of the encryption certificate."
         },
         "encryptionJwk": {
           "$ref": "#/definitions/EncryptionJwkResource",
-          "description": "The encryption cerfiticate key"
+          "description": "The encryption cerfiticate key."
```
Comment (BryanLa) [inline arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json]: Typo in "cerfiticate"

File: arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json [hunk 8]
```diff
@@ -1663,48 +1663,48 @@
...
         "latestPublishedMsiVersion": {
           "type": "string",
-          "description": "latest published version of the gateway install msi"
+          "description": "Latest published version of the gateway install msi."
         },
         "publishedTimeUtc": {
           "type": "string",
           "format": "date-time",
-          "description": "gateway install msi published time"
+          "description": "Gateway install msi published time."
```
Comment (BryanLa) [inline arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json]: Multiple occurrences of "msi"; should they be MSI, as in Microsoft Installer?

**Review by BryanLa — COMMENTED** [review]
@steved0x - let me know if you can make the changes before 5pm PST and I can get this merged for you.

**Comment by steved0x** [comment]
@BryanLa thanks for reviewing this, I fixed these issues.

**Pull request merged**